src/createSchemaForm.tsx

```diff
@@ -565,20 +653,21 @@ export function createTsForm<
     const renderedFields = renderFields(schema, props);
     const renderedFieldNodes = flattenRenderedElements(renderedFields);
     return (
-      <FormProvider {..._form}>
-        <ActualFormComponent {...formProps} onSubmit={submitFn}>
-          {renderBefore && renderBefore({ submit: submitFn })}
-          {CustomChildrenComponent ? (
-            <CustomChildrenComponent {...renderedFields} />
-          ) : (
-            renderedFieldNodes
-          )}
-          {renderAfter && renderAfter({ submit: submitFn })}
-        </ActualFormComponent>
-      </FormProvider>
+      <>
+        {renderBefore && renderBefore({})}
```

Comment:
i'm not sure accepting renderBefore and after Props is really necessary here. we can simply do them outside the FormFragment above and so could consumers of the component. I'll probably go ahead and do that to clean things up